animated_toolbox/css/scss/_module-toolbox-pill.scss

```
@@ -0,0 +1,189 @@
+.toolbox-wrapper {
+  position: relative;
+  top: 0;
+  left: 0;
+  width: 100%;
+
+  @include respond-screen($bp-md) {
+    position: fixed;
+    width: 322px;
+    top: 190px;
+    right: -240px;
+    left: auto;
+  }
+}
+
+.toolbox {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+
+  @include respond-screen($bp-md) {
+    position: absolute;
+  }
+}
+
+.toolbox-item {
+  list-style: none;
+  width: 33.33%;
+
+  @include respond-screen($bp-sm) {
+    width: 16.661%;
+  }
+
+  @include respond-screen($bp-md) {
+    width: 100%;
+  }
+
+  &:first-child .icon-text-wrapper {
+    border-radius: none;
+
+    @include respond-screen($bp-md) {
+      border-top-right-radius: 37px;
+      border-top-left-radius: 37px;
+    }
+  }
+
+  &:last-child .icon-text-wrapper {
+    border-radius: none;
+
+    @include respond-screen($bp-md) {
+      border-bottom-right-radius: 37px;
+      border-bottom-left-radius: 37px;
+    }
+  }
+
+  &:hover {
+    .toolbox-heading-wrapper {
+      transform: translateX(-285px);
+      opacity: 1;
+    }
+  }
+
+  &:nth-child(1) .toolbox-heading-wrapper {
+    animation-delay: 100ms;
+  }
+
+  &:nth-child(2) .toolbox-heading-wrapper {
+    animation-delay: 200ms;
+  }
+
+  &:nth-child(3) .toolbox-heading-wrapper {
+    animation-delay: 300ms;
+  }
+
+  &:nth-child(4) .toolbox-heading-wrapper {
+    animation-delay: 400ms;
+  }
+
+  &:nth-child(5) .toolbox-heading-wrapper {
+    animation-delay: 500ms;
+  }
+
+  &:nth-child(6) .toolbox-heading-wrapper {
+    animation-delay: 600ms;
+  }
+
+  &:nth-child(1) .toolbox-icon-svg {
+    animation-delay: 100ms;
+  }
+
+  &:nth-child(2) .toolbox-icon-svg {
+    animation-delay: 200ms;
+  }
+
+  &:nth-child(3) .toolbox-icon-svg {
+    animation-delay: 300ms;
+  }
+
+  &:nth-child(4) .toolbox-icon-svg {
+    animation-delay: 400ms;
+  }
+
+  &:nth-child(5) .toolbox-icon-svg {
+    animation-delay: 500ms;
+  }
+
+  &:nth-child(6) .toolbox-icon-svg {
+    animation-delay: 600ms;
+  }
```

- Please refactor to avoid unnecessary code duplication
- Easier to maintain

```suggestion
  &:nth-child(1) {

    .toolbox-heading-wrapper,
    .toolbox-icon-svg {
      animation-delay: 100ms;
    }
  }
  
  etc...
```